root/utility/tableColumns.js

```diff
@@ -91,6 +96,100 @@ export function defineTypeColumn(
   };
 }
 
+export function defineLabelCodeColumn(
+  order?: string = '',
+  sortable?: boolean = false,
+): ColumnOptions<{+label_code: number, ...}, number> {
+  return {
+    Cell: ({cell: {value}}) => (value
+      ? formatLabelCode(value)
+      : null),
+    Header: (sortable
+      ? (
+        <SortableTableHeader
+          label={l('Code')}
+          name="code"
+          order={order}
+        />
+      )
+      : l('Code')),
+    accessor: 'label_code',
+    id: 'label_code',
+  };
+}
+
+export function defineEntityColumn<D>(
+  getEntity: (D) => CoreEntityT,
+  columnName: string,
+  title: string,
+  order?: string = '',
+  sortable?: boolean = false,
+): ColumnOptions<D, string> {
+  return {
+    Cell: ({row: {original}}) => (getEntity(original)
+      ? <DescriptiveLink entity={getEntity(original)} />
+      : null),
```

Comment:
I'd add the extra couple lines here and store `getEntity(original)` in a variable so we don't have to call it twice.

Reply:
This would still need to be inside the Cell function, right? So something like this?: 

```
    Cell: ({row: {original}}) => {
      const entity = getEntity(original);
      return (entity
        ? <DescriptiveLink entity={entity} />
        : null),
```

Reply:
right!